runtime/description-dom-formatter.js

```diff
@@ -83,7 +92,16 @@ export default class DescriptionDomFormatter extends DescriptionFormatter {
         template = template.concat(`${index == 0 && i == 0 ? token.text[0].toUpperCase() + token.text.slice(1) : token.text}`);
       } else {  // view or slot handle.
         let sanitizedFullName = token.fullName.replace(/[.{}_\$]/g, '');
-        template = template.concat(`<span>{{${sanitizedFullName}}}</span>`);
+        let attribute = '';
+        if (i == 0) {
+          // Capitalize the first letter in the token.
+          template = template.concat(`<style>
+            [firstletter]::first-letter { text-transform: capitalize; }
```

Comment:
nit: this is actually a lot more expensive than just capitalizing the data directly in the model.

Reply:
added a TODO - i don't resolve the data in the model until later, and right now i don't have a way to remember to capitalize it then.